director/projects/views/mixins.py

```diff
@@ -204,6 +204,17 @@ def get_object(self, *args, **kwargs):
         )
         return self.project_fetch_result.project
 
+    def is_permitted(
```

Comment:
There are a few places where `self.has_permission` is called in existing code. Can you put TODOs in to use this shortcut function if possible. Otherwise we end up with inconsistencies.

Reply:
Instead of adding TODO's I went and changed 2 of the 5 instances of `has_permission` to `is_permitted` - I wasn't sure if it was appropriate for the other 3.